stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -208,6 +214,96 @@ public void run() {
     });
   }
 
+  @ChromeDevtoolsMethod
+  public PerformSearchResponse performSearch(JsonRpcPeer peer, JSONObject params) {
+    final PerformSearchRequest request = mObjectMapper.convertValue(
+        params,
+        PerformSearchRequest.class);
+
+    // Searches are case insensitive so convert
+    // the query to lower case for simpler comparisons.
+    request.query = request.query.toLowerCase();
```

Comment:
prefer not overwriting the query field in the request. instead, bounce to a local variable and use that, e.g.
```
final String requestQuery = request.query.toLowerCase();
```
(fuller explanation available upon request!)